PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/NativeAdUnit.java

```diff
@@ -165,4 +165,9 @@ public void addAsset(NativeAsset asset) {
         params.addAsset(asset);
     }
 
+    public String getCacheId() {
```

Comment:
A native ad unit might be requested multiple times. this is not a good design. Let's sync offline.

Reply:
Right, we can sync on that or can opt for the alternate solution.